src/pages/authentication/sign-up/Sign-up.css

```diff
@@ -0,0 +1,23 @@
+.modal > .card-content * {
+  margin: 8px 0;
+}
+.wrapper_modal > .modal_title {
+  padding: 0;
+  border-bottom: 1px solid var(--secondary-li);
+}
+.btn_modal > .close-modal {
```

Comment:
Generally should avoid complex css selectors

Reply:
I will try to avoid them!